mjpc/norm.cc

```diff
@@ -14,11 +14,11 @@
 
 #include "mjpc/norm.h"
 
+#include <mujoco/mujoco.h>
```

Comment:
I'm not sure why your linter is switching the header order -- let's not do that.

Reply:
I think the formatter considers `<mujoco/mujoco.h>` to be a C system header. [Google C++ style: Names and Order of Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
reverted to the previous ordering